controllers/category.controller.js

```diff
@@ -0,0 +1,76 @@
+const db = require('../DB/connection');
+
+exports.getAll = async (req, res) => {
+    try {
+        const userId = 1
+        // const userId = req.user.id;
+        const [categories] = await db.query('SELECT * FROM categories WHERE user_id = ?', [userId]);
+
+        const result = categories.map(category => ({
```

Comment:
Create a new folder called decorators, there creates a new file called category.decorator.js. There create a method with this method, finally use that decorator on every endpoint on this file.

```
const categoryDecorator = category => ({id: category.id,
            name: category.name
        }));
```